main.go

```diff
@@ -239,18 +243,25 @@ func runCLIApp(c *cli.Context) (err error) {
 
 	logger.Infof("Start gcsfuse/%s for app %q using mount point: %s\n", getVersion(), flags.AppName, mountPoint)
 
-	flagsStringified, err := util.Stringify(*flags)
-	if err != nil {
-		logger.Warnf("failed to stringify cli flags: %v", err)
-	} else {
-		logger.Infof("GCSFuse mount command flags: %s", flagsStringified)
-	}
+	// Log mount-config and the CLI flags in the log-file.
+	// If there is no log-file, then log these to stdout.
+	// Do not log these in stdout in case of daemonized run
+	// if these are already being logged into a log-file, otherwise
+	// there will be duplicate logs for these in both places (stdout and log-file).
+	if flags.Foreground || mountConfig.LogConfig.FilePath == "" {
+		flagsStringified, err := util.Stringify(*flags)
+		if err != nil {
+			logger.Warnf("failed to stringify cli flags: %v", err)
+		} else {
+			logger.Infof("GCSFuse mount command flags: %s", flagsStringified)
+		}
 
-	mountConfigStringified, err := util.Stringify(*mountConfig)
-	if err != nil {
-		logger.Warnf("failed to stringify config-file: %v", err)
-	} else {
-		logger.Infof("GCSFuse mount config flags: %s", mountConfigStringified)
+		mountConfigStringified, err := util.Stringify(*mountConfig)
+		if err != nil {
+			logger.Warnf("failed to stringify config-file: %v", err)
```

Comment:
same here, should we add gcsfuse here in statement in error message as well ?

Reply:
@ankitaluthra1  The current change is mostly just refactoring. The change to add `GCSFuse` in logs is needed many other places as well, such as in `SuccessfulMountMessage` and erros. So, let's do it in a separate PR after discussing and finalizing.

Reply:
yes got it..